partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnBetween.kt

```diff
@@ -206,9 +206,9 @@ internal val Fn_BETWEEN__TIME_TIME_TIME__BOOL = Function.static(
     ),
 
 ) { args ->
-    val value = args[0].time
-    val lower = args[1].time
-    val upper = args[2].time
+    val value = args[0].localTime
+    val lower = args[1].localTime
+    val upper = args[2].localTime
```

Comment:
Seems like we're missing function definitions for timez and timestampz? Will those be added at some point? Similarly for the other builtin functions?